development/contribute-to-docs.md

```diff
@@ -0,0 +1,148 @@
+
+At any time, you can see which files have been edited with the following git command:
+
+```bash
+git status
+```
+
+### Adding a new item to the page structure
+
+Often, you may want to add a new file to Timescale docs. To do so, you must 
+edit the `page-index/page-index.js` file. You should be able to glean 
+which lines to add and how to modify them based on the existing structure of 
+the file. If you need help, let us know in the 
+[Timescale community Slack][timescale-slack].
+
+### Submitting a pull request
+
+A pull request is a method of submitting contributions to an open development 
+project. It occurs when a developer asks for changes committed to an external 
+repository to be considered for inclusion in a project's main repository after 
+the review.
+
+Once you’ve completed all of your edits and are ready to submit a pull request, 
+you will first need to add all your edited files to your branch:
+
+```bash
+git add .
+```
+
+Now, you need to commit your changes along with a message explaining what changes 
+you’ve made to the repository. Your commit message should start with an active verb
+and longer descriptions should be in the body of your commit message.
+
+```bash
+git commit -m “Add a better description of what a feature does”
+```
+
+Then you have to upload your changes to GitHub:
+
+```bash
+git push origin my-branch-name
+```
+
+You will then need to complete the process on the 
+[Timescale Docs GitHub repository][timescale-docs-github]. Click on 
+the `New Pull Request` button and follow the instructions to create a 
+pull request using the branch you uploaded.
+
+### Making changes to a pull request
+
+Typically, you’ll be asked to make some changes to your pull request. You 
+can do so locally and re-add your changes to your branch and re-upload 
+it using the following commands:
+
+```bash
+git add .
+git commit --amend
+git push -f origin my-branch-name
+```
+
+Note that using `git amend` and force pushing are our preferred methods 
+because this is content and not code.
+
+Then go back to the specific pull request page in the 
+[Timescale Docs GitHub repository][timescale-docs-github] 
+and request another review by clicking the “cycle” button next 
+to the reviewer’s name.
+
+You may go through several such review cycles, which is normal.
+
+### Wrapping up
+
+After your pull request has been approved, you will have to go to the
+Pull Requests tab on the [Timescale Docs GitHub repository][timescale-docs-github]
+and click the "Rebase and merge" button.
+
+Once you’re done and your pull request has been merged, you can clean up 
+your workspace by first going back to the master branch and re-fetching the 
+repository as it is stored on GitHub. This will not only get all changes 
+you’ve made, but also all changes other people have made to the repository.
+
+```bash
+git checkout master
+git fetch origin && git reset --hard origin/master && git clean -f -d
```

Comment:
1. I don't know that you want to instruct folks to reset their master branch, _especially_ as part of a command with git fetch.  Probably `git pull` is more appropriate, with or without the `--rebase` flag.
2. Why are you advocating using `git clean`?